### DISCUSSION TITLE
compression challenge hardware

### DISCUSSION THREAD

**Discussion by conwayz** [commaai/commavq#30] [General]
Hi, are there any details on what hardware the submissions will be run on, e.g. is there GPU available? Also is there a time limit for the decompression script?

**Comment by conwayz** [comment]
@YassineYousfi are you able to advise? please feel free to redirect if this is not the correct forum

**Comment by YassineYousfi** [comment]
We can run the decompression script on a machine with 8x4090 GPUs
There is no formal time limit. But anything reasonable is fine (a few hours is fine, a few days is not fine)

**Reply by conwayz** [reply]
great, thanks!
